controllers/machine_controller.go

```diff
@@ -348,6 +349,11 @@ func (r *MachineReconciler) drainNode(cluster *clusterv1.Cluster, nodeName strin
 		DryRun: false,
 	}
 
+	if noderefutil.IsNodeUnreachable(node) {
+		// When the node is unreachable and some pods are not evicted for as long as this timeout, we ignore them.
```

Comment:
We should probably include a logging message here e.g "This node is unreachable, draining will wait for pod deletion during `skipWaitForDeleteTimeoutSeconds` after the request and will skip otherwise." cc @ncdc @alexander-demichev @hypnoglow

Reply:
I can open a PR if everyone is ok with the change.

Reply:
Do we want this to be something end users are aware of? If so, I would suggest an Event in addition to the log message.